style.css

```diff
@@ -0,0 +1,102 @@
+:root{
+    --color-1 : rgb(33, 37, 41);
+    --color-2 : rgb(52, 58, 64);
+    --color-3 : rgb(73, 80, 87);
+    --color-4 : rgb(108, 117, 125);
+    --color-5 : rgb(206, 212, 218);
+    --color-6 : rgb(233, 236, 239);
```

Comment:
Variable naming could be better here. You can use words like- primary, secondary, accent, border, background etc. Much more understandable.